pkg/scheduler/core/spreadconstraint/group_clusters_test.go

```diff
@@ -219,3 +220,162 @@ func Test_GroupClustersWithScore(t *testing.T) {
 		})
 	}
 }
+
+var duplicated = "duplicated"
+var aggregated = "aggregated"
+var dynamicWeight = "dynamicWeight"
+var staticWeight = "staticWeight"
+
+type RbSpecMap map[string]*workv1alpha2.ResourceBindingSpec
+
+func generateRbSpec(replica int32) RbSpecMap {
+	rbspecDuplicated := &workv1alpha2.ResourceBindingSpec{
+		Replicas: replica,
+		Placement: &policyv1alpha1.Placement{
+			ReplicaScheduling: &policyv1alpha1.ReplicaSchedulingStrategy{
+				ReplicaSchedulingType: policyv1alpha1.ReplicaSchedulingTypeDuplicated,
+			},
+		},
+	}
+	rbspecAggregated := &workv1alpha2.ResourceBindingSpec{
+		Replicas: replica,
+		Placement: &policyv1alpha1.Placement{
+			ReplicaScheduling: &policyv1alpha1.ReplicaSchedulingStrategy{
+				ReplicaSchedulingType:     policyv1alpha1.ReplicaSchedulingTypeDivided,
+				ReplicaDivisionPreference: policyv1alpha1.ReplicaDivisionPreferenceAggregated,
+			},
+		},
+	}
+	rbspecDynamicWeight := &workv1alpha2.ResourceBindingSpec{
+		Replicas: replica,
+		Placement: &policyv1alpha1.Placement{
+			ReplicaScheduling: &policyv1alpha1.ReplicaSchedulingStrategy{
+				ReplicaSchedulingType:     policyv1alpha1.ReplicaSchedulingTypeDivided,
+				ReplicaDivisionPreference: policyv1alpha1.ReplicaDivisionPreferenceWeighted,
+				WeightPreference: &policyv1alpha1.ClusterPreferences{
+					DynamicWeight: "AvailableReplicas",
+				},
+			},
+		},
+	}
+	rbspecStaticWeight := &workv1alpha2.ResourceBindingSpec{
+		Replicas: replica,
+		Placement: &policyv1alpha1.Placement{
+			ReplicaScheduling: &policyv1alpha1.ReplicaSchedulingStrategy{
+				ReplicaSchedulingType:     policyv1alpha1.ReplicaSchedulingTypeDivided,
+				ReplicaDivisionPreference: policyv1alpha1.ReplicaDivisionPreferenceWeighted,
+				WeightPreference: &policyv1alpha1.ClusterPreferences{
+					StaticWeightList: []policyv1alpha1.StaticClusterWeight{},
+				},
+			},
+		},
+	}
+	return RbSpecMap{
+		"duplicated":    rbspecDuplicated,
+		"aggregated":    rbspecAggregated,
+		"dynamicWeight": rbspecDynamicWeight,
+		"staticWeight":  rbspecStaticWeight,
+	}
+}
+
+func generateClusterScores(n int, scores []int64, replicas []int64) []ClusterDetailInfo {
+	info := make([]ClusterDetailInfo, n)
+	for i := 0; i < n; i++ {
+		info[i] = ClusterDetailInfo{
+			Name:              fmt.Sprintf("member%d", i+1),
+			Score:             scores[i],
+			AvailableReplicas: replicas[i],
+		}
+	}
+	return info
+}
+
+type GroupScoreArgs struct {
+	id          int
+	clusters1   []ClusterDetailInfo
+	clusters2   []ClusterDetailInfo
+	rbSpec      *workv1alpha2.ResourceBindingSpec
+	minGroups   int
+	want        bool
+	description string
+	// if clusters1 is better than clusters2, want is true
```

Comment:
Is this the comment forgot to remove? Or put in the wrong place?

Reply:
I have put it in the correct place @RainbowMango